plugins/sync-font/host.js

```diff
@@ -1,11 +1,13 @@
 export function hostSyncFont(host) {
 	host.onRequest('font', () => {
-		const computedStyle = window.getComputedStyle(document.body);
+		const bodyComputedStyle = window.getComputedStyle(document.body);
+		const rootComputedStyle = window.getComputedStyle(document.documentElement);
 		const visualRedesign = document.body.classList.contains('visual-redesign');
+
 		return {
-			family: computedStyle.fontFamily,
-			size: computedStyle.fontSize,
-			visualRedesign: visualRedesign
+			family: bodyComputedStyle.fontFamily,
```

Comment:
Still using body to grab the font family. Isn't used but have it for compatibility.